torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -757,6 +768,8 @@ class TORCH_API ProcessGroupNCCL : public Backend {
   // If all comms on this PG are fully initialized, return true.
   bool isInitialized();
 
+  ErrorType getError();
+
```

Comment:
Same suggestion for uplifting. For non-NCCL backends, we can leave it as unimplemented.

Reply:
Was debating about this with myself too, can make it in any backend